var/spack/repos/builtin/packages/blis/package.py

```diff
@@ -111,8 +111,12 @@ def darwin_fix(self):
 
     @property
     def libs(self):
+        shared = True if '+shared' in self.spec else False
```

Comment:
Could simplify:

```suggestion
        shared = '+shared' in self.spec
```

Reply:
Removed this line and used shared = '+shared' in self.spec within return find_libraries block.